hbw/selection/common.py

```diff
@@ -234,8 +234,72 @@ def sl_boosted_jet_selection(
 )
 
 
+@selector(
+    exposed=False,
+)
+def noise_filter(
+    self: Selector,
+    events: ak.Array,
+    results: SelectionResult,
+    **kwargs,
+) -> Tuple[ak.Array, SelectionResult]:
+    mask = ak.Array(np.ones(len(events), dtype=bool))
+    for flag in self.noise_filter:
+        mask = mask & events.Flag[flag]
+
+    results.steps["noise_filter"] = mask
+    return events, results
+
```

Comment:
As far as I know the MET filters are implemented in CF directly and one can add them via the config. Is there a reason to write an own function?

Reply:
Removing the line for now is the best solution I guess.